### ISSUE TITLE
add -R flag to read from files previously created by geco_data

### ISSUE THREAD

**Issue by stefco** [stefco/geco_data#11]
equivalent to the `-T` flag, but will look for output files matching the given parameters and will read straight from those in order to maximize speed.

**Comment by stefco** [comment]
- [x] check the header of each file to make sure it matches the specified channel and time range before concatenating it to the output

**Comment by stefco** [comment]
doooooooonnnnne and it is fast:

``` bash
[stefan.countryman@ldas-pcdev2 geco_data]$ time geco_dump -T -s 'Nov 1 2015' -e 'Nov 1 00:10:00 2015' H1:SYS-TIMING_Y_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_3 > /dev/null

real    0m3.208s
user    0m2.435s
sys 0m0.657s
[stefan.countryman@ldas-pcdev2 geco_data]$ time geco_dump -R -s 'Nov 1 2015' -e 'Nov 1 00:10:00 2015' H1:SYS-TIMING_Y_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_3 > /dev/null

real    0m0.048s
user    0m0.012s
sys 0m0.027s
```

**Comment by stefco** [comment]
around 200x faster

**Comment by stefco** [comment]
from that one example

**Comment by stefco** [comment]
implemented in fe24cd5
